example/src/scenes/GroupedList.js

```diff
@@ -9,6 +9,7 @@ import type { Theme } from 'react-native-ios-kit/types';
 
 type Props = {
   theme: Theme,
+  //eslint-disable-next-line
```

Comment:
I think we should implement correct type here instead of ignoring the lint error.

If it's too much work, make sure to specify which lint error you're ignoring in the comment, and a short reason